module3/labs/Week2/RestfulRoutes.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: page
+title: RESTful Routes Lab
+---
+
+### Background
+
+Today we're going to start building an application called `Record Collection` that we'll keep building on throughout this mod.
+
+Start by cloning [this](https://github.com/turingschool-examples/RecordCollectionStarter) repo: **Do not fork!** We might add new branches for you to check-out in the future and we are trying to avoid having many copies of the same repo. Then checkout the branch `RESTful-routes-starter`.
+
+To get the app running, you will want to follow a few steps.
+1. Run `Update-Database` to apply the migration and create your DB
+2. Run some SQL in pgAdmin or create a SeedData file to seed some data

```

Comment:
Should we have a minimum number of records they should be creating? Like 3 albums minimum?